typespec-tests/tsp/response.tsp

```diff
@@ -157,4 +184,13 @@ interface ResponseOp {
   // OperationDetails does not have "result" property
   @route("/lro-invalid-result")
   lroInvalidResult is CustomLongRunningOperation<Resource, OperationDetails2>;
+
+  @route("/paged-string")
+  listStrings(): StringsList;
+
+  @route("/paged-int32")
+  listIntegers(): Int32sList;
+
+  // @route("/paged-datetime")
+  // listDateTimes(): DateTimesList;
```

Comment:
This does not work at present, will handle in a follow-up PR.

It would require certain non-trival conversion from `BinaryData` to e.g. `OffsetDateTime` or `Duration`.